sdk/search/azure-search/src/main/java/com/azure/search/DataSources.java

```diff
@@ -56,34 +56,37 @@ public static DataSource azureSql(String name, String sqlConnectionString, Strin
     /**
      * Creates a new DataSource to connect to an Azure SQL database.
      *
-     * @param name The name of the data source.
+     * @param dataSourceName The dataSourceName of the data source.
      * @param sqlConnectionString The connection string for the Azure SQL database.
-     * @param tableOrViewName The name of the table or view from which to read rows.
+     * @param tableOrViewName The dataSourceName of the table or view from which to read rows.
```

Comment:
Looks like a find and replace error.

```suggestion
     * @param tableOrViewName The name of the table or view from which to read rows.
```